tests/cypress/integration/components/warning-callout.js

```diff
@@ -0,0 +1,29 @@
+describe('warning call out component tests', function () {
+  beforeEach(function () {
+    cy.viewport('macbook-15');
+  });
+
+  it('Navigating to warning callout Component', function () {
+    cy.visit(Cypress.config().baseUrl + '/warning-callout.html')
+  });
+
+  it('checking for warning callout class', function () {
+    cy.get('.nhsuk-warning-callout')
+  });
+
+  it('checking for warning callout label class', function () {
+    cy.get('.nhsuk-warning-callout').children().should('have.class', 'nhsuk-warning-callout__label')
+  });
+
+  it('Checking background colour of the label', function () {
```

Comment:
We don't need to check the background colour, BackstopJS can do that.

Reply:
Backstop checks whether the colour matches with the reference, but not the colour gradient - rgb(255, 235, 59)

Reply:
But what is the purpose of checking the colour gradient? We aren't checking the colour contrast ratio for accessibility purpose so its a pointless test no? BackstopJS will pick up if the colour changes.